.github/workflows/maven.yml

```diff
@@ -0,0 +1,23 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ualibweb/*]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 17
+        uses: actions/setup-java@v2
+        with:
+          java-version: "17"
+          distribution: "temurin"
+          cache: maven
+      - name: Build with Maven
+        run: mvn -B verify --file pom.xml
```

Comment:
Jenkins -- this was really only added for Bama's use before we had access to the Jenkins CI and other utilities.  I'll remove this since the Jenkins workflow is much more comprehensive and consistent with other FOLIO modules.